app/src/main/java/corp/kairos/adamastor/Statistics/StatisticsAppsMenuAdapter.java

```diff
@@ -28,10 +29,7 @@ public StatisticsAppsMenuAdapter(Context context, Set<AppDetails> apps) {
         super(context, R.layout.activity_allapps);
         this.context = context;
         this.apps = apps;
-        finalTotal = 0;
-        for(AppDetails app : apps) {
-            finalTotal += app.getUsageStatistics();
-        }
+        this.finalTotal = apps.iterator().hasNext() ? apps.iterator().next().getUsageStatistics() : 0L;
```

Comment:
Can you please explain this line? Is this meant to be equivalent to what you had before? i.e.: 
```JAVA
for(AppDetails app : apps) {	
    finalTotal += app.getUsageStatistics();	
}
```

Reply:
The apps are order by time of usage. This variable is now having the time of application that was used the most, in other words, the first of the apps set. This is just a way to get, but I can refact this if it is not clear what I mean.

Reply:
Fix it, see if it its more clean now.